### DISCUSSION TITLE
Problem of geometric field oversize

### DISCUSSION THREAD

**Discussion by MINE0126** [lululxvi/deepxde#939] [Q&A] [answered]
Hello, everyone, 
I want to use Deepxde to compute a problem with a relatively sizeable geometric domain (300m x 100m). Many training points and more extended training costs are required without linear scaling of the geometric field. I have used linear scaling for the geometric domain question (scaled down by a factor of 100).
Is there any relevant paper that discusses whether it is correct and feasible to perform the linear scaling method in this way?
Thanks.

**Comment by lululxvi** [comment, marked as answer]
- https://en.wikipedia.org/wiki/Dimensional_analysis
- https://en.wikipedia.org/wiki/Change_of_variables
